raidprotect/src/event/message/handle.rs

```diff
@@ -14,5 +21,41 @@ pub async fn handle_message(message: Message, state: Arc<ClusterState>) {
     let parsed = parse_message(&message);
     state.redis().set(&parsed).await.ok();
 
+    if is_an_old_command(&message) {
+        if let Err(error) = warn_about_old_command(&message, state.http()).await {
+            error!(error = ?error, "failed to warn user about question mark commands deprecation");
+        }
+    }
+
     info!("received message: {}", message.content) // Debug util real implementation
 }
+
+async fn warn_about_old_command(message: &Message, http: &Client) -> Result<()> {
+    let lang = Lang::fallback();
```

Comment:
The lang can be obtained from the message author.

```suggestion
    let lang = message.author.locale.map(Lang::from).unwrap_or_else(|| Lang::fallback());
```